srcpkgs/discord-ptb/template

```diff
@@ -1,19 +1,31 @@
 # Template file for 'discord-ptb'
-pkgname="discord-ptb"
-version=0.0.9
+pkgname=discord-ptb
+version=0.0.21
 revision=1
+archs="x86_64"
 wrksrc="DiscordPTB"
 depends="alsa-lib dbus-glib gtk+3 GConf libnotify nss libXtst libcxx libatomic"
-archs="x86_64"
-nopie=yes
-repository=nonfree
-restricted=yes
 short_desc="Proprietary freeware VoIP application"
 maintainer="Abel Graham <abel@abelgraham.com>"
-license="proprietary"
+license="custom:Proprietary"
 homepage="https://discordapp.com/"
 distfiles="https://dl-ptb.discordapp.net/apps/linux/${version}/discord-ptb-${version}.tar.gz"
-checksum=372d6f75203626370218c97c21519b779e5af334177cafa65a5655e357c77520
+checksum=a1fa4e20cce5f493d84c9e3c7ac74c8265768886859e98022fa41d102f2726c4
+_license_checksum=e3b0c44298fc1c149afbf4c8996fb92427ae41e4649b934ca495991b7852b855
```

Comment:
This is checksum of empty string. Empty file is produced because of
```
=> discord-ptb-0.0.21_1: running post_extract ...
/void-packages/srcpkgs/discord-ptb/template: line 21: w3m: command not found
/void-packages/srcpkgs/discord-ptb/template: line 20: curl: command not found
```

Add w3m to hostmakedepends, use `$XBPS_FETCH_CMD -o eula` instead of `curl -L`.

Reply:
I've installed both, so the errors weren't being thrown for me. Thanks for checking it out!

Reply:
license in package is still empty

Reply:
@Chocimier If you have any ideas on how to fix it, I'm welcoming any. I'm not very knowledgable and I don't know much about VoidLinux, but this above code works for me.

Please help me fix this template so we can merge this.

Reply:
`$XBPS_FETCH_CMD -o eula` outputs to eula file, not stdout. Do something like `$XBPS_FETCH_CMD -o eula; cat eula | ...`, rebuild (with local revbump possibly to create new package, but then send with revision=1) and check if packaged license has right content.

Reply:
I might be thick-skulled, but I'm not exactly sure why we don't want to save it to a file, but instead output it to a stdout. Is there a reason for it?

I'll try doing the above as soon as my laptop is free(few hours).

Reply:
I figured out that I was being thick-skulled and I realised what the problem with the file was...
Piping didn't do anything if the output of the first command wasn't outputted to the stdout.

In the meantime, there has been an update to 0.0.22, so I'll update to that and commit again.